dev/integration_tests/ui/test_driver/keyboard_resize_test.dart

```diff
@@ -33,7 +33,7 @@ void main() {
 
       bool heightTextDidShrink = false;
       for (int i = 0; i < 3; ++i) {
-        await Future<void>.delayed(const Duration(seconds: 1));
+        await Future<void>.delayed(const Duration(seconds: 3));
```

Comment:
Is there an explicit signal we could wait for rather than a delay?

Bumping the delay out may work today but still leaves room for future flakes and if we can't wait for an explicit signal, it might mean our APIs are insufficient.

Reply:
Api 30 added new apis around window resizing that could possibly help. 
https://medium.com/androiddevelopers/animating-your-keyboard-fb776a8fb66d
https://yatmanwong.medium.com/android-how-to-pan-the-page-up-more-25fc5c542a97

I will add that this test already does loop twice and this change modifies the step taken. If we are going to increase the time I think we should modify line 35 to 6 instead of increasing the step duration.

Reply:
> Is there an explicit signal we could wait for rather than a delay?
> 
> Bumping the delay out may work today but still leaves room for future flakes and if we can't wait for an explicit signal, it might mean our APIs are insufficient.

Looks like the framework does not directly expose a way to check this; we would have to wait for the bottom inset height to change to detect this in one way or another. Looks like Android doesn't directly give a way to check for keyboard visibility either, unless I'm missing something.

Going to land this since the test is flaking pretty frequently. @johnmccutchan if you think this is worth pouring more effort into, let me know; more than happy to file an issue for finding a longer term/more fool-proof solution!